### ISSUE TITLE
Test to sync Asana with GH but not vice versa

### ISSUE THREAD

**Issue by Mttbnchtt** [SDG-InterfaceOntology/sdgio#138]
I create this issue to test whether I correctly sync-ed Asana with GH but not vice versa.

**Comment by Mttbnchtt** [comment]
I created this issue. Then I sync-ed GH and Asana (both directions). This issue appeared in Asana in the column To Do and To Sync with GH. Then I deleted this issue in Asana and I sync-ed Asana to GH (but not vice versa).  As desired, the issue was not deleted in GH. I will now proceed to delete another issue in Asana that has been copied from GH. The result should be the same. If it is, I will later delete all the foreign issues from Asana so that they will be present only in GH.
